src/EditorFeatures/CSharpTest/Diagnostics/UseImplicitOrExplicitType/UseImplicitTypeTests.cs

```diff
@@ -1082,7 +1082,7 @@ static void M()
         [WpfFact, Trait(Traits.Feature, Traits.Features.CodeActionsUseImplicitType)]
         public async Task SuggestVarOnFrameworkTypeEquivalentToBuiltInType()
```

Comment:
:bulb: Based on the previous method, this test should actually be named `DoNotSuggestVarOnFrameworkTypeEquivalentToBuiltInType_WithOption`.